primefaces/src/main/resources/META-INF/resources/primefaces/validation/validation.validators.js

```diff
@@ -132,11 +132,13 @@ if (window.PrimeFaces) {
 
                 for (var file of value) {
                     if (allowtypesRegExp && (!allowtypesRegExp.test(file.type) && !allowtypesRegExp.test(file.name)))  {
-                        throw vc.getMessage(this.ALLOW_TYPES_MESSAGE_ID, file.name);
+                        throw vc.getMessage(this.ALLOW_TYPES_MESSAGE_ID, file.name,
```

Comment:
unnecessary cosmetic change

Reply:
dear @tandraschko, 

this and some others you have listed are surely not cosmetic but required changes. here, for example, a further argument was added to extend the display of the validation error messages. i admit that there were also a few "cosmetic" changes included in this PR, which were simply done in "batch" mode to improve the readability of the code. Not worth to spend any comments on it. Instead, a little more appreciation for the contribution would be helpful, as I always appreciate your and the communities great work.

many thanks in advance.

Reply:
1. fully agreed. i should have made several PRs here.
2. in principle yes, but the next time you do a search & replace of a certain pattern again, you will have to look again into all the cases instead of quickly replacing them all consistently. 
3. it would never come into my mind any form of disrespect when I fix a typo by search & replace or do a tiny reformat to avoid unneeded newlines as in the following case:

```
          assertEquals("Invalid file size.",
                page.messages.getMessage(0).getSummary());
to 
          assertEquals("Invalid file size.", page.messages.getMessage(0).getSummary());
```

would be great if can review and merge this PR.
many thanks in advance.